website/content/api-docs/secret/pki.mdx

```diff
@@ -1023,6 +1023,11 @@ when signing an externally-owned intermediate.
    Access to this endpoint should be restricted by policy to only trusted
    operators.
 
+~> **Note**: As of 1.17.x the default behavior has changed to truncate the
+   signed intermediary's notAfter field to the signing issuer's notAfter
+   if it is computed to go beyond. Prior versions permitted the notAfter
+   to go beyond.
```

Comment:
```suggestion
By default, Vault truncates the `notAfter` field of the signed intermediary
to use the `notAfter` field of the signing issuer, it the computed field for the
intermediary goes beyond the prescribed length.
```
The docs are versioned, so calling out the version isn't necessary and making the text an aside doesn't necessarily add meaningful context, so we can just make the information part of the document.